supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -13,6 +13,16 @@
 // Fuse: Prefixed second sentence with "In Red Hat Fuse, an EIP is a" (Breda)
 // Fuse: Changed "Camel" to "Red Hat Fuse" (Breda)
 // Fuse: Removed the final sentence about the Palette (Breda)
+
+[[egress]]
+==== image:images/yes.png[yes] egress (adjective, noun)
+*Description*: In RedHat OpenShift, _egress_ describes the outgoing direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
```

Comment:
```suggestion
*Description*: In RedHat OpenShift, _egress_ describes the outgoing direction of network traffic. For example, traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
```

Reply:
I like Brian's suggestion as well, so i'm going to combine these two. Thank you!

Reply:
Also I saw your comment about parts of speech, and I agree I think I can remove the noun bc I removed the references to specific OpenShift product area names. This definition is really just an adjective now. For the Ingress definition, I think we settled on using (noun, adjective) instead of creating a separate entry. But I think there's not many others that do that.